guides.md

```diff
@@ -28,6 +29,145 @@ our values.
+    _:gear:(top-left) > Mail menu item > Forward tab_
+8. Check "Forward incoming mail" and enter provided primary email, clicking "Keep a copy".
+9. From your own @hypha.com email, send a welcome email, like this template:
+    >
+    > Hi XXXX --
+    >
+    > Yay! Your Hypha email's set up and forwarding to your personal inbox. People can start emailing you at this address immediately, without any extra effort on your part.
+    >
+    > Your new email (and login): XXX@hypha.coop  
+    > Your password: Sent via Signal to XXX-XXX-XXXX
+    >
+    > Having said that, when you'd like to start _sending_ email as XXX@hypha.coop, you'll need to take some more steps.
+    >
+    > The next steps will depend on how you like to handle email:
+    > - [If you'd like to use it from your existing Gmail account...]()
+    > - [If you'd like to use it from our hosted webmail interface...]()
+    > - [If you'd like to use it from another email client...]()
+    >
+    > Once you're set up, let us know by sending a test email to infra@hypha.coop, ideally with a short emoji-only story. (Keypad mashes a-ok!) We'll send a friendly poke in a week if we haven't heard back.
+    >
+    > Questions? Feedback? Reach out via infra@hypha.coop or [`#hyphacoop-infra-ops:tomesh.net`](https://chat.tomesh.net/#/room/#hyphacoop-infra-ops:tomesh.net) for chat.
+    >
+    > In Solidarity,  
+    > xxxxx
+10. Send a low-entropy Signal message with the previously noted password, for example: `🍄 password: xxxxxxxx`
+
+### Using your new inbox (Users)
+
+Accessing your inbox can be done via the hosted webmail interface or by using the same desktop email client you're already using.
+
+What you'll need:
+- Your new email address
+- Your password (sent to you by admin on initial setup)
+
+#### Using GMail
+
+1. Log into GMail
+2. Go to GMail settings, by navigating:  
+    _:gear: (top-right) > Settings > Accounts and Import_
+3. Under "Send mail as", click "Add another email address".
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of GMail settings page" src="https://i.imgur.com/oQMdIS9.png" />
+    </details>
+5. In the new popup, enter:
+    - _Name_: as will appear to email recipients
+    - _Email address_: new email address e.g., `handle@hypha.coop`
+    - _Treat as alias_: checked :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of new alias popup" src="https://i.imgur.com/I2RGv80.png" />
+    </details>
+4. Click "Next", then:
+    - _SMTP Server_: `mailninja.aseriesoftubez.com` (auto-filled)
+    - _Port_: `587` (auto-filled)
+    - _Username_: replace with new email address, e.g., `handle@hypha.coop`
+    - _Password_: your password
+    - TLS: checked :ballot_box_with_check:
+    <details>
+      <summary>View Screenshot</summary>
+      <img alt="Screenshot of new alias popup" src="https://i.imgur.com/rh3YhK4.png" />
+    </details>
+5. Click "Add Account".
```

Comment:
For buttons/text in the UI, I usually put upticks like `Add Account`. I don't know if we should do that vs. "quotes" or **bold** but I think we should stay consistent.

Reply:
I think we should keep **bold** for emphasis, so it's really a choice between `upticks` and "quotes" for UI.

<img width="773" alt="Screen Shot 2020-03-17 at 10 38 10 AM" src="https://user-images.githubusercontent.com/2261308/76843149-a8a58380-683b-11ea-849d-a378a1bfe7bf.png">

Curious if @dcwalk has thoughts since she authored a large part of the handbook as well.

Reply:
What if we just say "let's keep each guide self-consistent" and leave it at that?

Reply:
If I'm the only one who does `upticks` I rather change all my guides to "quotes". I prefer site-wide consistency, it's much easier for readers.

Reply:
For me:

Agreed that bold is best for emphasis. Italics could be just for "notes". _Note: For things like this._ Quotes for field names and wayfinding things on screen. `backticks` for inputs and interactions, but not necessarily help text or descriptions of them.

Wayfinding (clicking through menus) should have something besides quotes, since they get awkward when there's a list of quoted items.

The above is mostly how I've been thinking during writing (might have lost some of the above when co-writing with elon).

Reply:
We could write a sample guide to encompass all the above in a condensed space, instead of writing a "guide for guides"? But I'd rather it be like a template, and no one will be prodded if they choose to do their own thing for whatever reason.

Reply:
>Wayfinding (clicking through menus) should have something besides quotes, since they get awkward when there's a list of quoted items.

Agree and I use `backticks` for wayfinding, but also UI elements like the `Submit` button.

Reply:
I'll let you guys decide what to do here. Still would like @dcwalk's opinion on formatting preferences.

Reply:
Agree that we need to be consistent across the docs on these things, having it shift even on the same page, or in different sections is confusing and will tire out the reader trying to follow along.

Can we spawn an issue for style guide and not block this here?

Reply:
Initial thoughts on styling in #62